README.md

```diff
@@ -1 +1,53 @@
 # Fisher's Manager README
+
+## How I operate
+
+### Feedback
+
+1. How do you like feedback - the medium (Slack, email, in-person, etc.)
+> Personally, I like feedback in-person. I'm also open to having tricky conversations over Slack, but when I said I want to take it offline, I ask that you respect that.
+2. How do you like feedback - routine like in 1:1s, or as-it-happens
+> Timely feedback is always better for me -- just grab me!
+3. How do you prefer to receive recognition? (public or private)
+> I'm totally fine with public recognition but don't expect it!
+
+### Goals and Support
+
+1. What makes 1:1s the most valuable for you?
+> 1:1s are a time for me to voice what I'm worried about.
+2. What are your goals for this year? And for the next three months?
+> In my role as an engineering leadership sponsor I want to continue to improve our back-of-house systems' reliability and simplicity and support Casper's growth in the wholesale 
+3. What do you need from your manager?
+> I need help solving problems. I don't need help tracking against goals -- I just want clear expectations so I know where I stand and I want help solving specific problems that I come to you with. I will often start a conversation with "Here's what I'm looking for...".
+4. From your team? (this one is good if the 1:1 is with someone who manages people, but is typically relevant for IC’s, too)
+> I need clear communication (both up and down) -- tell me when you're struggling. Tell me when a deadline is slipping. Tell me when you need more support. And tell me when to back off.
+5. From your peers? (outside the team)
+> I expect clear and timely feedback -- if you disagree with my approach, or an action I took, tell me as soon as possible!
+
+### Expectations of folks who report to me
+
+Please invest in your 1:1s -- it's your time. You own your 1:1, so it is my expectation is for you to come with a topic to discuss. Here are some ideas:
+1. What are you worried about?
```

Comment:
It's funny to hear `worry` as the focus of 1:1's in both directions. It's realistic I suppose, but does sound a bit negative. Also interesting that this is a longer list of things than what you bring to your own 1:1's.